core/src/main/java/io/substrait/type/proto/BaseProtoTypes.java

```diff
@@ -22,6 +22,13 @@ abstract class BaseProtoTypes<T, I> {
   public final T INTERVAL_DAY;
   public final T INTERVAL_YEAR;
   public final T UUID;
+  public final T VARCHAR;
+  public final T FIXED_CHAR;
+  public final T FIXED_BINARY;
+  public final T DECIMAL;
+  public final T MAP;
+  public final T LIST;
+  public final T STRUCT;
```

Comment:
Timestamp was recently deprecated in favor of PrecisionTimestamp.  Should we add it here as well (although it additionally has the precision which might be a similar issue to decimal.